packages/frontend/src/hooks/useFetchedPhotos/index.jsx

```diff
@@ -1,10 +1,16 @@
 import { useEffect, useState } from 'react';
 
-const useFetchedPhotos = ({ query = null, perPage }) => {
+const useFetchedPhotos = ({ query = null, perPage, storedPhotos }) => {
```

Comment:
Instead of adding a new variable here, could the photos value from the photos context be used instead? The `useState` function in the hook would be replaced with the logic to access the value from the photos context instead.

Reply:
@perennialAutodidact I'm having trouble understanding this change and I'm wondering if you could talk it out with me a bit. 

The use fetched photos hook does not have access to the context because it's not a child of the photos provider. 

In my head, the new variable is useful to differentiate between the fetched photos returned by the hook and the photos that are already saved in the context.  When the useFetchedPhotos hook is called in the photos provider, it returns the new photos and is passed as a prop the photos currently saved in the context.